deep_water_level/src/deep_water_level/infer.py

```diff
@@ -75,10 +75,10 @@ def run_dataset_inference(
     model,
     dataset_dir: Path,
     annotations_file: str,
-    normalized_output,
-    crop_box = None,
-    use_water_line = False,
```

Comment:
@adamantivm do you have format-on-save enabled? I think this file might not have been formatted last time you commited.

Reply:
Would you remind me how to enable it?

Reply:
In your user settings
```
   "[python]": {
        "editor.formatOnSave": true
    },
```